.github/workflows/run-analysis.yml

```diff
@@ -0,0 +1,35 @@
+name: Run Analysis
+
+# Controls when the action will run.
+on:
+  workflow_dispatch:
+  pull_request:
+
+jobs:
+  build_and_run:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        name: Checkout repo
+        with:
+          fetch-depth: 0
+          ref: ${{ github.head_ref }}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Build
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          push: false
+          tags: openpedcan-analysis:latest
+
+      - name: Image digest
+        run: echo ${{ steps.docker_build.outputs.digest }}
+
+      - name: Run analysis in container
+        run: ./scripts/run-analysis.sh bash download-data.sh
```

Comment:
I think the working directory is correct. If we see any error in this step, it could be the directories are not aligned.

Assuming the current working directory is the cloned repository directory, say `~/OpenPedCan-analysis`, `./scripts/run-analysis.sh` is accessible which bind mounts `--volume "$(pwd)":/rocker-build/`. As `/rocker-build` is the working directory of the docker image, the docker run will have `download-data.sh` in current working directory.

https://github.com/PediatricOpenTargets/OpenPedCan-analysis/blob/b430cbb2d7f50cc4d9f7675395e487b03c16834b/Dockerfile#L442